src/content/en/2019/third-parties.md

```diff
@@ -0,0 +1,167 @@
+
+The remainder of third-party providers also tend to fall into this usage pattern. Whether it’s hosting of a utility library such as jQuery for usage by front-end developers cached on Cloudflare’s edge servers or a vast library of common fonts served from a popular Google CDN, third-party content is another way to give the site owner one fewer thing to worry about and, maybe, just maybe make the job of delivering a great experience a little bit easier.
+
+## Repercussions
+
+### Performance
+
+The performance impact of third-party content is neither categorically good or bad. There are good and bad actors across the spectrum and different category types have varying levels of influence.
+
+The good: shared third-party font and stylesheet utilities are, on average, delivered more efficiently than their first-party counterparts.
+
+Utilities, CDNs, and Content categories are the brightest spots on the third-party performance landscape. They offer optimized versions of the same sort of content that would otherwise be served from first-party sources. Google Fonts and Typekit serve optimized fonts that are smaller on average than first-party fonts, Cloudflare CDN serves a minified version of open source libraries that might be accidentally served in development mode by some site owners, Google Maps SDK efficiently delivers complex maps that might otherwise be naively shipped as large images.
+
+The bad: a very small set of entities represent a very large chunk of JavaScript execution time carrying out narrow set of functionality on pages.
+
+Ads, social, hosting, and certain analytics providers represent the largest negative impact on web performance. While hosting providers deliver a majority of a site’s content and will understandably have a larger performance impact than other third-party categories, they also serve almost entirely static sites that demand very little JavaScript in most cases that should not justify the volume of script execution time. The other categories hurting performance though have even less of an excuse. They fill very narrow roles on each page they appear on and yet quickly take over a majority of resources.
+
+The silver lining of such a small number of entities enjoying such large market share is that a very limited and concentrated effort can have an enormous impact on the web as a whole. Performance improvements at just the top few hosting providers improve 2-3% of all web requests.
+
+### Privacy
+
+The abundance of analytics providers and top heavy concentration of script execution raises two primary privacy concerns for site visitors: site owners are more interested in tracking their users than any other third-party use case and a handful of companies receive information on a large swatch web traffic.
```

Comment:
What does "on a large swatch web traffic" mean? Did you mean: "on a large swath of web traffic"?

Reply:
oh yes I did indeed great typo catch!

Reply:
Apparently *swatch* is also a viable alternative in some cases. Learned something new haha